pubnub-kotlin/pubnub-kotlin-api/src/iosMain/kotlin/com/pubnub/api/Endpoint.ios.kt

```diff
@@ -19,3 +23,7 @@ fun <T> Consumer<Result<T>>.onFailureHandler(mapper: (NSError?) -> Throwable = {
         accept(Result.failure(mapper(error)))
     }
 }
+
+fun String.toNSData(): NSData? {
```

Comment:
What's the best place for helper methods like this?

Reply:
you could create a file in `com.pubnub.kmp` package, for example `converters.kt`, and put some top-level functions there